src/main/java/ch/cern/eam/wshub/core/services/contractmanagement/EquipmentReservationAdjustmentService.java

```diff
@@ -0,0 +1,23 @@
+package ch.cern.eam.wshub.core.services.contractmanagement;
+
+import ch.cern.eam.wshub.core.annotations.Operation;
+import ch.cern.eam.wshub.core.client.InforContext;
+import ch.cern.eam.wshub.core.interceptors.LogDataReferenceType;
+import ch.cern.eam.wshub.core.services.INFOR_OPERATION;
+import ch.cern.eam.wshub.core.services.contractmanagement.entities.EquipmentReservationAdjustment;
+import ch.cern.eam.wshub.core.tools.InforException;
+
+public interface EquipmentReservationAdjustmentService {
+
+    @Operation(logOperation = INFOR_OPERATION.COMMENT_C, logDataReference1 = LogDataReferenceType.INPUTFIELD, logDataReference1FieldName = "entityKeyCode", logDataReference2 = LogDataReferenceType.RESULT)
```

Comment:
The INFOR_OPERATION is used on the logger, so it should be a unique string per operation. Create new constants for these 4.

Reply:
logDataReference1FieldName - Is "entityKeyCode" existing on the DTO?